Source/WebInspectorUI/UserInterface/Views/ColorSwatchDisplay.js

```diff
@@ -0,0 +1,60 @@
+WI.ColorSwatchDisplay = class ColorSwatchDisplay extends WI.Object
```

Comment:
I tend to agree with @dcrousso here that this could be inlined into `WI.ColorPicker`. 
There's little that `WI.ColorSwatchDisplay` does that justifies its own file and it's very tightly coupled with WI.ColorPicker as the delegate anyway.

Aside: it would be nice if there's a way to extract the sorting of colors as a standalone utility function that can be tested independently of the view.

Reply:
Yeah I agree, I created it initially as its own class because I envisioned this class expanding to hold sorting features and more. But recently after talking to Razvan about including a new model and such, I agree it should be inlined. I will do that

Reply:
i thought the plan was to inline this inside `WI.ColorPicker`?